merchants/tests/test_pruning.py

```diff
@@ -0,0 +1,151 @@
+from datetime import datetime, timedelta
+from unittest import mock
+from uuid import uuid4
+
+from assertpy import assert_that
+from django.test import TestCase
+from django.utils.timezone import get_default_timezone
+from freezegun import freeze_time
+
+from merchants.models import PaymentStatus
+from merchants.tests.factories import PaymentOrderFactory
+from merchants.verkkokauppa.order.exceptions import CancelOrderError
+from merchants.verkkokauppa.payment.exceptions import GetPaymentError
+from merchants.verkkokauppa.payment.test.factories import PaymentFactory
+from merchants.verkkokauppa.payment.types import PaymentStatus as WebShopPaymentStatus
+from reservations.models import STATE_CHOICES
+from reservations.tests.factories import ReservationFactory
+
+from ..pruning import update_expired_orders
+
+TIMEZONE = get_default_timezone()
+
+
+class UpdateExpiredOrderTestCase(TestCase):
```

Comment:
What I see here is using the freeze_time first as "base time" and then doing some action after some time has passed. I think we could add the freeze time as a decorator like;
```
@freeze_time(datetime(2022, 11, 28, 10, 10, 0, tzinfo=TIMEZONE))
class UpdateExpiredOrderTestCase(TestCase):
```
and then could remove the first `with freeze_time...` from the tests.


Note that this is fine as is now and no action needed. Surely I'm not even sure how the freeze time acts when you've used the decorator and then use it with the `with` statement